wqflask/wqflask/snp_browser/snp_browser.py

```diff
@@ -4,7 +4,7 @@
 from PIL import (Image)
 
 from utility.logger import getLogger
-logger = getLogger(__name__ )
+logger = getLogger(__name__)
```

Comment:
**praise:** +1 for removing the extra space. I(or @zsloan) should probably do a grep to fix this globally :smile: